packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -215,12 +215,17 @@ export const CallComposite = (props: CallCompositeProps): JSX.Element => {
   } = props;
   useEffect(() => {
     (async () => {
-      await adapter.askDevicePermission({ video: true, audio: true });
-      adapter.queryCameras();
-      adapter.queryMicrophones();
-      adapter.querySpeakers();
+      if (role === 'Consumer') {
```

Comment:
I guess we need to conditionally compile this part?

Reply:
Good catch. Yes we do!